src/simple_continuous_time_system.py

```diff
@@ -1,3 +1,49 @@
+#!/usr/bin/env python
+
+import numpy as np
 import pydrake
+from pydrake.systems.framework import (DiagramBuilder, VectorSystem)
+from pydrake.systems.primitives import SignalLogger
+from pydrake.systems.analysis import Simulator
+import matplotlib.pyplot as plt
+
+class SimpleContinuousTimeSystem(VectorSystem):
+    def __init__(self):
+        VectorSystem.__init__(self,
+            0,                           # Zero inputs.
+            1)                           # One output.
+        self._DeclareContinuousState(1)  # One state variable.
+
+    # xdot = -x + x^3
+    def _DoCalcVectorTimeDerivatives(self, context, input, state, derivatives):
+        derivatives[0] = -state[0] + state[0]**3
```

Comment:
What about something closer to the math symbols, like:
```
def _DoCalcVectorTimeDerivatives(self, context, u, x, xd):
    xd[:] = -x + x**3
```
That could put it closer to the math?

Reply:
I debated that w/ @jwnimmer-tri for the C++ version.  perhaps the right answer could be different here.

Reply:
i need to make sure i understand why that works...
why do I need the [:]?
and the **3 is defined for the numpy array, too?

Reply:
Yeah, the `[:]` is a little confusing, but it allows for value assignment, since you don't really have `operator=` overloads in Python.